packages/transformer/src/IModelExporter.ts

```diff
@@ -438,14 +448,21 @@ export class IModelExporter {
       return;
     }
 
-    const startChangeset =
-      args && "startChangeset" in args ? args.startChangeset : undefined;
+    const isEmptyObject = (obj: object): boolean =>
+      Object.keys(obj).length === 0;
 
-    const initOpts: ExporterInitOptions = {
-      startChangeset: { id: startChangeset?.id },
-    };
+    let initOpts: ExporterInitOptions;
```

Comment:
I am unsure why this was added. I suspect Nick stumbled into some issue similar to the one I also reported: https://github.com/iTwin/imodel-transformer/issues/236

We should consider if this is really the right approach for this problem. My first instinct is that we should rather save processed version and use that as default value, instead of "process the current changeset of the source iModel being exported when startChangeset.id is undefined.". I am not sure if this change means only last (current) change set would be processed? In such case this could result in unpredictable behavior if there was more than 1 change set added since last transformation.

Reply:
I remember it now, this was related to:  https://github.com/iTwin/imodel-transformer/issues/241